src/javascript/app_2/Modules/Reports/Containers/statement.jsx

```diff
@@ -0,0 +1,107 @@
+import { PropTypes as MobxPropTypes }       from 'mobx-react';
+import PropTypes                            from 'prop-types';
+import React                                from 'react';
+import { withRouter }                       from 'react-router-dom';
+import { localize }                         from '_common/localize';
+import DataTable                            from 'App/Components/Elements/DataTable';
+import { getContractPath }                  from 'App/Components/Routes/helpers';
+import { connect }                          from 'Stores/connect';
+import { getStatementTableColumnsTemplate } from '../Constants/data-table-constants';
+import PlaceholderComponent                 from '../Components/placeholder-component.jsx';
+import { ReportsMeta }                      from '../Components/reports-meta.jsx';
+import EmptyTradeHistoryMessage             from '../Components/empty-trade-history-message.jsx';
+
+class Statement extends React.Component {
+    componentDidMount()    { this.props.onMount(); }
+    componentWillUnmount() { this.props.onUnmount(); }
+
+    getRowAction = (row_obj) => {
+        let action;
+
+        if (row_obj.id && ['buy', 'sell'].includes(row_obj.action_type)) {
+            action = getContractPath(row_obj.id);
+        } else if (['deposit', 'withdrawal'].includes(row_obj.action_type)) {
+            action = {
+                message: row_obj.desc,
+            };
+        }
+
+        return action;
```

Comment:
Instead of creating a variable then returning that why not we just do this?

```
getRowAction = row_obj => {
    if (row_obj.id && ['buy', 'sell'].includes(row_obj.action_type)) {
         return getContractPath(row_obj.id);
    }

    if (['deposit', 'withdrawal'].includes(row_obj.action_type))  {
         return { message: row_obj.desc };
    }
}
```

Reply:
@ahkuma because then you get this:
```Expected to return a value at the end of arrow function. (consistent-return)```

Basically, it needs to `return` something one way or another, not just sometimes. (Yes, even if the thing it returns is an `undefined` variable)